catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/model/Registry.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define([
+    'wreqr',
+    'js/model/Service.js',
+    'js/model/RemoteStatus.js',
+    'backbone',
+    'underscore',
+    'poller',
+    'js/model/Status.js',
+    'backboneassociation'
+],
+function (wreqr, Service, RemoteStatus, Backbone, _, poller, Status) {
+    var Registry = {};
+
+
+    Registry.ConfigurationList = Backbone.Collection.extend({
+        model: Service.Configuration
+    });
+
+    Registry.Model = Backbone.Model.extend({
```

Comment:
Might want to use `Backbone.AssociatedModel` since this technically has another model (the Registry.ConfigurationList) inside of it.